cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -63,6 +63,8 @@ public final class DataprocUtils {
   public static final String CDAP_GCS_ROOT = "cdap-job";
   public static final String CDAP_CACHED_ARTIFACTS = "cached-artifacts";
   public static final String WORKER_CPU_PREFIX = "Up to";
+  // The property name for disabling caching of artifacts in GCS uploaded to GCS Bucket used by Dataproc.
+  // // It can be overridden by profile runtime arguments (system.profile.properties.disableGCSCaching)
   public static final String DISABLE_GCS_CACHING = "disableGCSCaching";
```

Comment:
Generally I would favor positive config (e.g. `enable.gcs.caching`) instead of negated one.

Reply:
GCS Caching is enabled by default so we thought of keeping it `disableGCSCaching` rather than `enableGCSCaching` which makes it sound more intuitive for disabling the cache. Does it make sense??

Reply:
ack